src/components/Tokens.ts

```diff
@@ -10,13 +10,13 @@ const Tokens: Array<Token> = [
     {
         imgPath: "/crypto/eth.svg",
         symbol: "ETH",
-        address: "0xbb6f4D4544aB61840226ae0694F737752a69449B",
+        address: process.env.NEXT_PUBLIC_TOKEN_A_ADDR || '',
```

Comment:
This was not working on my end as it was throwing me error. Most probably because that token was not deployed on hardhat. So, if you are going to be on this page it will also throw the error that's why I added this so that we do not have the problem. And I think once the factory is ready we will be fetching the token data from there so we won't need this then